### PR TITLE
Add checkbox color

### PR DESCRIPTION
**Pull request by KeyWeeUsr**
Add color property to checkbox, for easier working with dark/gray backgrounds. Working example in the `checkbox.py`

### CODE REVIEW

File: kivy/uix/checkbox.py [hunk 1]
```diff
@@ -142,6 +142,21 @@ class CheckBox(ToggleButtonBehavior, Widget):
     'atlas://data/images/defaulttheme/checkbox_radio_disabled_on'.
     '''
 
+    color = ListProperty([1,1,1,1])
```
Comment (kived) [inline kivy/uix/checkbox.py]: E231: missing whitespace after ','

File: kivy/uix/checkbox.py [hunk 2]
```diff
@@ -142,6 +142,21 @@ class CheckBox(ToggleButtonBehavior, Widget):
...
+
+    Color is in the format (r, g, b, a). Use alpha greater than 1 for
+    brighter colors. Alpha greater than 4 causes blending border and check
+    mark together.
+
+    .. versionadded:: 1.9.2
+
+    :attr:`color` is a
+    :class:`~kivy.properties.ListProperty` and defaults to
+    '[1,1,1,1]'.
```
Comment (kived) [inline kivy/uix/checkbox.py]: PEP8 checker probably won't catch this in a docstring, but same thing here.

File: kivy/uix/checkbox.py [hunk 3]
```diff
@@ -153,3 +168,16 @@ def _toggle_active(self):
...
         self.state = 'down' if value else 'normal'
+
+
+if __name__ == '__main__':
+    from kivy.base import runTouchApp
+    from kivy.uix.gridlayout import GridLayout
+    box = GridLayout(cols=3)
+    box.add_widget(CheckBox())
+    box.add_widget(CheckBox(group='g0'))
+    box.add_widget(CheckBox(group='g0', color=[1,1,1,2]))
```
Comment (kived) [inline kivy/uix/checkbox.py]: And here... ;)

**Comment by KeyWeeUsr** [comment]
@kived Forgot about pep8. Repaired :)

File: kivy/uix/checkbox.py [hunk 4]
```diff
@@ -142,6 +142,21 @@ class CheckBox(ToggleButtonBehavior, Widget):
     'atlas://data/images/defaulttheme/checkbox_radio_disabled_on'.
     '''
 
+    color = ListProperty([1, 1, 1, 1])
+    '''Color is used for tinting the default graphical representation
+    of checkbox and radio button(images).
```
Comment (dessant) [inline kivy/uix/checkbox.py]: Space before the left parenthesis. :mag:

**Comment by dessant** [comment]
Thanks a bunch. :+1:

**Pull request merged**